### DISCUSSION TITLE
Change language default and date format

### DISCUSSION THREAD

**Discussion by rushian** [Shelex/cypress-allure-plugin#140] [Q&A] [answered]
Hi, 

I would like to set the default language to Br, and change the date format [to dd/MM/yyyy]; how do i do this?

I am using "@shelex/cypress-allure-plugin": "^2.26.5" and "allure-commandline": "^2.17.2"

**Comment by Shelex** [comment, marked as answer]
Hi @rushian .
Unfortunately there is no option to influence that from plugin. 
I have also found an issue opened for allure: https://github.com/allure-framework/allure2/issues/952
There is a comment with a workaround where you need to build your own allure binary, or you can just vote for that issue.

**Reply by rushian** [reply]
thank you for help...
